modules/ding_app_variables/ding_app_variables.module

```diff
@@ -50,3 +57,34 @@ function ding_app_variables_display() {
     'carousels' => $carousel_configs,
   );
 }
+
+/**
+ * Get panels configurations.
+ *
+ * @param string $carousel
+ *   Pass subtype of panel pane.
+ *
+ * @return array
+ *   List of carousels pane configurations.
+ */
+function _ding_app_variables_get_carousel_configs($carousel) {
+  $query = db_select('panels_pane', 'pp');
+  $result = $query
```

Comment:
I think that it would be a bette solution to load the configuration through the panels "API" as suggested by @holt83 

This would ensure all use-cases and be a more robust solution.

@kasperg be free to comment if you disagree whit this.